### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by UlhasD** [Alterhu/CucumberPeople#17]
Not working on my Mac OS X , Eclipse Luna 4.4.2 
issue is as per installation instructions, I have installed the plugin but after restart 
there are no Cucumber listed in wizard

**Comment by Alterhu** [comment]
please tag or untag this option when install it from eclipse update:"Group items by category" 
![2015-09-08_235603](https://cloud.githubusercontent.com/assets/5029046/9755235/35051a8e-5685-11e5-93bb-54343d605811.png)

**Comment by scottboyd-ai** [comment]
This doesn't work.

**Comment by Alterhu** [comment]
Yes, i need to finalize a new release for Mac machine .sorry for this
convenience ...

On Tue, Nov 10, 2015 at 10:10 AM, scottboyd-ai <notifications@github.com>
wrote:

> This doesn't work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Alterhu/CucumberPeople/issues/17#issuecomment-155517900>
> .
>



-- 
*Best Regards*
* Alter.hu*
